web-template/src/main/java/org/ehrbase/webtemplate/parser/EnhancedAqlPath.java

```diff
@@ -225,6 +235,24 @@ public void addOtherPredicate(String name, String value) {
     otherPredicates.put(name, value);
```

Comment:
Getting rid of this method would render the class immutable.
This should allow for additional performance enhancements etc.